src/app/job-registration-form/job-registration-form.component.html

```diff
@@ -0,0 +1,208 @@
+<form class="jobForm" [formGroup]="jobForm" (ngSubmit)="addJob()">
+    <div class="example-container">
+        <h1>Job registration</h1>
+
+        <div class="job-form-field">
+            <mat-form-field appearance="outline">
+                <mat-label>Idea</mat-label>
+                <input formControlName="idea" matInput placeholder="" required maxlength="64" />
```

Comment:
you should put field max length values into constants

Reply:
const JOB_REGISTRATION_FORM_OPTIONS = {
  ideaMaxLength: 64,
 organizationMaxLength: 64,
 ....ect
}

Reply:
+